topic04-git-introduction/book-git-tutorial/01.Basics.md

```diff
@@ -0,0 +1,30 @@
+# **_Git guide to commands_**
+
+This manual contains necessary commands for git when working on your own, contributing to other projects and working as a team.
+
+###### What is git about?

```

Comment:
would maybe use a larger heading here, so the jump from '#' to '#####' is not so large